core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -555,6 +568,17 @@ private case class OffsetsToClean(firstDirtyOffset: Long,
 }
 
 private[log] object LogCleanerManager extends Logging {
+  private val UncleanablePartitionsCountMetricName = "uncleanable-partitions-count"
```

Comment:
Is there a reason why decided to have these variable names start with a capital letter?

Reply:
> Is there a reason why decided to have these variable names start with a capital letter?

https://github.com/apache/kafka/pull/13623#discussion_r1182593025